generators/beagle/src/main/scala/TestHarness.scala

```diff
@@ -75,15 +75,18 @@ class BeagleTestHarnessInner(implicit p: Parameters) extends LazyModule
 
     dut.reset := reset
     dut.boot := true.B
-    dut.alt_clks.foreach { _ := clock }
-    dut.alt_clk_sel := 0.U
+    dut.single_clks.foreach { _ := clock }
```

Comment:
Also worth maybe making one of them faster and one of them slower. Maybe the single_clk is 2x slower than diff clock?